python/producer/kafka_producer_config.py

```diff
@@ -0,0 +1,42 @@
+import os
+
+KAFKA_PREFIX = 'KAFKA_'
+STRIMZI_PREFIX = 'STRIMZI'
+
+# Strimzi environment variables declaration
+TOPIC = 'STRIMZI_TOPIC'
+MESSAGE = 'STRIMZI_MESSAGE'
+
+# default values for Strimzi environment variables
+DEFAULT_TOPIC = 'my-topic'
+DEFAULT_MESSAGE = 'Hello to the World'
+
+# default values for environmental variables in dictionary
+DEFAULT_PROPERTIES = {
+    'bootstrap.servers': 'localhost:9092'
+}
+
+
+# convert the kafka (strimzi) prefix
```

Comment:
Might be worth giving a more explicit description of what this function does e.g. Converts env var names to variable names that are compliant with Kafka properties file format

Reply:
Done